develop/devguide/Connector/Howto/How_to_handle_XML_and_JSON_data_in_QActions.md

```diff
@@ -44,7 +44,7 @@ To generate C# classes from JSON text, you can rely on a built-in Visual Studio
 Alternatively, there are many online converters to automatically convert JSON text into an object-oriented C# class. Here are a few examples:
 
 - <https://JSON2csharp.com/>
-- <https://JSONformatter.org/JSON-to-csharp>
+- <https://JSONformatter.org/>
```

Comment:
This link does (no longer) seem to support converting Json to C#. We could provide the following link instead: https://quicktype.io/csharp